pkg/registry/apis/provisioning/repository/local.go

```diff
@@ -112,35 +112,34 @@ func (r *localRepository) Config() *provisioning.Repository {
 }
 
 // Validate implements provisioning.Repository.
-func (r *localRepository) Validate() (fields field.ErrorList) {
+func (r *localRepository) Validate() field.ErrorList {
```

Comment:
I don't think we should fallthrough when checking if the path is not good for a reason.